FMI-CROSS-CHECK-RULES.md

```diff
@@ -1,83 +1,56 @@
+Instead of long tables and horizontal scrollbars, this dynamic visualization encourages users to search, filter and zoom to explore the data which generally fits on a single page.
 Users can even bookmark specific search, filter and selection settings and/or email their results to others as a single URL.
 
 If you find any issues, please report them in the [FMI cross-check repository](https://github.com/modelica/fmi-cross-check/issues).
 
-Existing results and test FMUs provided in the repository that do not comply with the new rules were removed from the cross-check repository.
+Existing results and test FMUs provided in the repository that do not comply with new rules were removed from the cross-check repository.
 
 Vendors are encouraged to fix these excluded results and test FMUs.
 
-## 5.	New Display of the Cross-Check Results
+## 5.	Display of the Cross-Check Results
 
-The new FMI Cross-Check infrastructure now displays tools and their status as well as individual results more dynamically.
-These new Cross-Check Rules will not define how to show these results anymore.
+The FMI Cross-Check infrastructure displays tools and their status as well as individual results dynamically.
+These Cross-Check Rules will not define how to show these results anymore.
 The FMI Steering Committee may decide at any time to change the way these results are displayed to improve readability and show different levels of compliance.
 
 
-## 6. New Fee structure for listing Tools at the FMI Cross-Check Site
-
-The FMI project requires funds to help improve infrastructure and to run administration more professionally.
-
-- It was decided to introduce an annual listing fee per tool of **2000,-- &euro;**.
-
--  If there are meaningful Cross-Check results published on the web-site (tool is shown with green status), the annual tool fee is reduced to **1000,-- &euro;**.
--  **This fee is waived for tools that are [open source](https://opensource.org/osd)**.
-
--  The FMI MAP reserves the right to change these fees and their structure from year to year.
-
--  Fees are bound to a calendar year.
-
--  The listing fee will not entitle tool vendors to influence how the results are being displayed.
-
-The FMI Steering Committee is sole decision maker for changes to the display mechanisms.
-If vendors do not agree how the results are displayed, vendors can ask at any time for removal of their data.
-Fees are non-refundable in any case.
+## ~~ 6. Fee structure for listing Tools at the FMI Cross-Check Site ~~
 
+Previous plans to charge a listing fee for tools on the Cross-Check site have been dropped by the FMI Steering Committee.
 
 ## 7. Reference FMUs
 
-In addition to FMUs exported by tools listed in the tools table, there will be a (growing) number of Reference FMUs.
+In addition to FMUs exported by tools listed in the tools table, there will be a (growing) number of [Reference FMUs](https://github.com/modelica/Reference-FMUs).
```

Comment:
"will be" --> "are"?

Reply:
We will have to have them automatically exported into or referenced in the XC (versioned, of cause).
@t-sommer : Do you have a plan there?

I have changed the text to "is" ("a growing number").